app/src/main/java/org/simple/clinic/home/patients/PatientsEffect.kt

```diff
@@ -10,8 +10,10 @@ object RefreshUserDetails : PatientsEffect()
 
 object LoadUser : PatientsEffect()
 
-object LoadDismissedApprovalStatus : PatientsEffect()
+object LoadInfoForShowingApprovalStatus : PatientsEffect()
```

Comment:
This commit could also have been split to first have `LoadInfoForShowingApprovalStatus` updated and then `ShowUserWasApproved` effect being handled.

Reply:
Ugh, sorry about this. Will try splitting it up.

Reply:
Done.